### ISSUE TITLE
Please replace "blacklist" with something more emotionally and psychologically safe

### ISSUE THREAD

**Issue by dandelion-abiding** [vector-im/riotX-android#1705]
Seeing as how this application is likely to become something of a public good, I'm really appreciative of the recent name change.  Encrypted communication platforms are built in a tradition that holds confidentiality and privacy as a human right.  Using more inviting and accessible language is wonderful not only from a marketing standpoint, but from an social ethics perspective as well.

I request that "blacklist" be replaced with something less charged.  I realize that there's a tech tradition of using this language, but as more and more people start to depend on Element for personal and political safety, I believe it's urgently important for the interface of a security focused application to use mentally and emotionally safe language as well, and to do so in as inclusive a manner as possible.

[Gitlab decided](https://gitlab.com/gitlab-org/gitlab/-/issues/7554) to go with "allowlist" and "denylist".

Story time: 

Imagine that supremacists were running around your city with semiautomatic weapons, threatening to set houses on fire, leaving gas cans in peoples' yards.  You've received death threats because of your skin color, or because of the skin color of someone in your family.  This situation is not hypothetical... this has been happening in many parts of the US over the last month.  I witnessed.

Okay, now for the hypothetical part:

A kind neighbor, a friend of a trusted friend, has invited you to shelter in their house for the night.  You're sleeping on the couch.  But there's just one thing that feels a little off:  they have this creepy old-timey black and white photo of a minstrel show hanging on the wall in the living room, across from where you're sleeping.  In the photo, a group of smiling white people wear black paint on their faces and pose in goofy, stereotyping gestures.  You can tell that your host has a kind heart, and is likely oblivious about how the painting might make you feel, and you have very little choice but to trust them in this situation, but the photo is very uncomfortable and triggers waves of anxiety.  You close your eyes and try to sleep.

I realized that's a pretty cartoonish situation, but I wanted to convey some urgency.  Having "blacklist" in a secure communication app's UI right now is absurd and unsettling in a way that's similar to the hypothetical situation I've just described.

**Comment by ktagbnaw** [comment]
See the origin of the term blacklist: https://en.wikipedia.org/wiki/Blacklisting#Origins_of_the_term

>Despite its origins and etymology, many incorrectly assume that the term has racial undertones, leading to controversies surrounding its usage.

**Comment by MSavoritias** [comment]
I don't think it really matters here if it really means that or where the term came from.

What mattters is that some members of our users/community don't like it. And since it doesn't costs us much to change it I don't see a reason not to. Besides I don't think any of us wants a random word that doesn't really change anything to be the hill they die on.

**Comment by HeyItsMeNobody** [comment]
> What mattters is that some members of our users/community don't like it. And since it doesn't costs us much to change it I don't see a reason not to. Besides I don't think any of us wants a random word that doesn't really change anything to be the hill they die on.

Look at the number of downvotes on your comment and the first comment, I think the members of our "community" don't like your idea. Besides, The term "blacklist" is used throughout the Matrix.org spec and can't just be changed on one client because it would cause confusion.

We can't just go around changing terms like "whitelist" and "blacklist" just because it includes the word "black", It's a color and the origin definitely matters. Stop being offended about normal words, No one cares.

Hope this reply doesn't come over as very rude or stupid, I'll delete it if that's the case.

**Comment by ZoomRmc** [comment]
In European tradition (and lots of others), "black" is often considered a colour with a negative load. This mostly stems from the fact that black is the colour of darkness, which has always been the state of increased unsafeness for our ancestors. It's a colour of the underworld in ancient Greece, it's everywhere in the Bible, it's a colour of mourning (at least Romans already used it for that matter).

On the other hand, black is the colour of strict, elegant style and authority. It's ultra-popular for clothing, design and interior decoration, due to its practicality, high contrast and ability to harmonize with other colours. No one thinks of a "little black dress" as of something that has the connotations I listed in the previous paragraph.

Personally, I very much doubt that breaking the language in this way will bring the desired effect and will be truly internalized, as it's going against very deep cultural roots which, in turn, have very logical and natural genesis.

Nothing there has anything to do with race or skin colour. 

PS: Denoting people by their skin tone is considered offensive in many countries. Speaking as a non-US citizen I can attest that there's a widespread confusion on why in American English there's a few words meaning *the same thing* but some of them are considered offensive and some are not, especially since these norms basically flipped during a single human lifespan.

**Comment by MSavoritias** [comment]
> > What mattters is that some members of our users/community don't like it. And since it doesn't costs us much to change it I don't see a reason not to. Besides I don't think any of us wants a random word that doesn't really change anything to be the hill they die on.
> 
> Look at the number of downvotes on your comment and the first comment, I think the members of our "community" don't like your idea. Besides, The term "blacklist" is used throughout the Matrix.org spec and can't just be changed on one client because it would cause confusion.
> 
> We can't just go around changing terms like "whitelist" and "blacklist" just because it includes the word "black", It's a color and the origin definitely matters. Stop being offended about normal words, No one cares.
> 
> Hope this reply doesn't come over as very rude or stupid, I'll delete it if that's the case.

We can see about the details of the implementation. If it is needed to change on the Matrix spec the issue can be raised there too.

As far as I saw there is one upvote in my comment and 10 positive reaction on the original post. The community seems divided. And it seems that people do care. Judging by here and the moves that are being done by RedHat and other companies.

As far as your remark for we "can't", I think it is a human language. By that I mean we can change it or adapt it into whatever we want. Its not something final that can't be changed.

The whole thing is being done for a minority that wants to feel included. Of course the downvotes are going to be more. People are hard to change. But also because the minority can't exactly vote the majoritarian way the problem. They will always lose. So the question is: Is the majority willing to change for the minority to be included?

Just because something is technically right doesn't mean that we can ignore the culture around it. Or the feelings and the reception people have to it.

**Comment by ara4n** [comment]
We have absolutely no attachment to the word blacklist; very happy to change it to blocklist or denylist or similar. (Going to lock this issue to contributors now, to avoid the temptation to over-debate the conclusion, which in the end is a call for the project to make.)
